### DISCUSSION TITLE
Assistance required for configuring OpenID Connect Plugin.

### DISCUSSION THREAD

**Discussion by Kalaivani19** [Kong/kong#13453] [Help] [answered]
Team – We are trying to implement the [OpenID Connect](https://docs.konghq.com/hub/kong-inc/openid-connect/) plugin; however, we were unable to locate the configuration for this plugin in the official [Kong Github](https://github.com/Kong/kong/tree/master/kong/plugins) repository. Would you be able to assist us with locating the configuration file for the plugin? Your help would be greatly appreciated.

**Comment by ProBrian** [comment, marked as answer]
Hi @Kalaivani19 , 
OpenID Connect is an enterprise plugin which is only available in Kong EE edition, so the code is not included in Kong CE edition([Kong Github](https://github.com/Kong/kong/tree/master/kong/plugins)).
